crates/optimism/node/src/node.rs

```diff
@@ -721,6 +724,19 @@ impl OpNetworkBuilder {
     }
 }
 
+fn get_all_op_bootnodes<Node>(ctx: &BuilderContext<Node>) -> Vec<Bootnode>
```

Comment:
```suggestion
pub fn get_all_op_bootnodes<Node>(ctx: &BuilderContext<Node>) -> Vec<Bootnode>
```
allow rollups to use this as they please